upstash_redis/commands.py

```diff
@@ -4229,6 +4230,256 @@ def script_load(self, script: str) -> ResponseT:
         return self.execute(command)
 
 
+class JsonCommands:
+    def __init__(self, client: Commands):
+        self.client = client
+
+    def arrappend(self, key: str, path: str, *value: JSONValueT) -> ResponseT:
+        """
+        Appends one or more values to a JSON array stored at a key.
+
+        Returns the length of the array after the append operation.
+
+        See https://redis.io/commands/json.arrappend
+        """
+        if len(value) == 0:
+            raise Exception("At least one value must be specified.")
+
```

Comment:
Let's not do these error checks in the client, and keep it dumb. applies to other places as well. this way, we can keep the error messages consistent